lib/slim-rails/register_engine.rb

```diff
@@ -0,0 +1,30 @@
+module Slim
+  module Rails
+    module RegisterEngine
+      def self.register_engine(app, config)
+        if ::Rails::VERSION::MAJOR == 3
+          register_engine3(app, config)
+        else
+          register_engine4(app, config)
```

Comment:
@hanachin nice, but you can safely rename register_engine4 to just register_engine since it also servers Rails 5 and maybe new releases (we'll deal with those when appear)